<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomFrameWindow</class>
 <widget class="QWidget" name="CustomFrameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QWidget" name="windowFrame" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#windowFrame
{
	border:1px solid transparent;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="WindowDragger" name="windowTitlebar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#windowTitlebar
{
	border: 0px none palette(high);
 	background-color:rgb(33, 34, 38);
 	height:20px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="titleText">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">  padding-left:5px;
  color:rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>Chat Client</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="minimizeButton">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#minimizeButton
{
	background-color:none;
	border:none;
	width:16px;
	height:16px;
	padding:4px;
	image:url(:/images/icon_window_minimize.png);
}

#minimizeButton:hover
{
	background-color: rgb(43, 44, 48);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="restoreButton">
           <property name="styleSheet">
            <string notr="true">#restoreButton
{
	background-color:none;
	border:none;
	width:16px;
 	height:16px;
 	padding:4px;
 	image:url(:/images/icon_window_restore.png);
}

#restoreButton:hover
{
	background-color: rgb(43, 44, 48);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="maximizeButton">
           <property name="styleSheet">
            <string notr="true">#maximizeButton
{
	background-color:none;
	border:none;
	width:16px;
	height:16px;
	padding:4px;
	image:url(:/images/icon_window_maximize.png);
}

#maximizeButton:hover
{
	background-color: rgb(43, 44, 48);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#closeButton
{
	background-color:none;
	border:none;
	width:16px;
	height:16px;
	padding:4px;
	image:url(:/images/icon_window_close.png);
	border-top-right-radius: 0px;
}

#closeButton:hover
{
	background-color:rgb(243, 71,71);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="windowContent" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#windowContent
{
	border: 0px none palette(base);
	border-radius:0px 0px 5px 5px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WindowDragger</class>
   <extends>QWidget</extends>
   <header>windowdragger.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
